src/wdio/services/TerraCommands/determine-test-options.js

```diff
@@ -49,7 +49,9 @@ const axeOptions = (args) => {
     options, viewports,
   } = determineArgs(args);
 
-  const axeRules = options.rules || options.axeRules;
+  const globalAxeRules = (global.browser.options.axe.options) ? global.browser.options.axe.options.rules : undefined;
+  const customAxeRules = options.rules || options.axeRules;
+  const axeRules = { ...globalAxeRules, ...customAxeRules };
```

Comment:
Does this work as expected? If I remember correctly, the glabal Axe rules will be passes as an array of rules config like:
```
axe: {
   options: {
        rules: [
            'rule': { enabled: false }
        ],
     }
}
```
 where as the custom axe rules are passes as an object of rules:
```
Terra.it.isAccessible({ rules: {
      'rule': { enabled: false },
 });
```

Reply:
@emilyrohrbough ,
Yes, you are right, the global axe rules and custom axe rules are different and the final merged rules contain both these formats with different key value pairs.
I might be wrong but i guess both the formats are supported, that is the reason why even after merging the rules with different formats the rules still apply.

Either way, I could only think of adding something similar to the following code to convert them into a single format:
```
const globalConfig = [ {id: 'color-contrast', enabled: false}, {id: 'aria-alt', enabled: true}];
const specConfig = { 'color-contrast': {enabled: true, any: 'hello'} };
const result = {};
globalConfig.forEach((entry, index) => {
  const {id, ...rest} = entry;
  result[id] = rest;
});
console.log({...specConfig, ...result});
```
So what are your thoughts or suggestions on this?

Reply:
Can we use array.reduce to accomplish the conversion from an array to an object?
We will also want to spread the test specific rules after the global config.

Have you tried a few options out and validated with tests since you've posted this question?

Reply:
I have implemented the same using `array.reduce` and spread the test specific rules after the global config.

> Have you tried a few options out and validated with tests

Yes, i have tried with a few options and the logic works but I have yet to write tests for this.